index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+      <div id="main">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <div id="img-div">
+          <div id="img-caption">
+            <h2>Image descriptiion</h2>
+          </div>
+          <img src="http://as00.epimg.net/img/comunes/fotos/fichas/equipos/large/3.png" alt="" id="image">
```

Comment:
A good practice is to write a description of the image on your alt attribute inside the img tag for accessibility reasons